etc/disable-common.inc

```diff
@@ -143,3 +143,8 @@ blacklist ${PATH}/xfce4-terminal
 blacklist ${PATH}/xfce4-terminal.wrapper
 blacklist ${PATH}/mate-terminal
 blacklist ${PATH}/mate-terminal.wrapper
+blacklist ${PATH}/lilyterm
+blacklist ${PATH}/pantheon-terminal
+blacklist ${PATH}/roxterm
+blacklist ${PATH}/roxterm-config
+blacklist ${PATH}/terminix
```

Comment:
There is also /usr/bin/urxvtcd, which is a wrapper (shell script) around urxvtc, which starts a server process if none is running. I don't think this one has to be added.